pydash/pydash_app/dashboard/aggregator/statistics.py

```diff
@@ -136,3 +134,104 @@ def append(self, endpoint_call, dependencies):
 
     def rendered_value(self):
         return date_dict({k: len(v) for k, v in self.value.items()})
+
+
+class FastestExecutionTime(Statistic):
+    dependencies = []
+
+    def empty(self):
+        return -1
+
+    def field_name(self):
+        return 'fastest_measured_execution_time'
+
+    def append(self, endpoint_call, dependencies):
+        if self.value < 0 or self.value > endpoint_call.execution_time:
+            self.value = endpoint_call.execution_time
+
+    def rendered_value(self):
+        return self.value
+
+
+class SlowestExecutionTime(Statistic):
+    dependencies = []
+
+    def empty(self):
+        return -1
+
+    def field_name(self):
+        return 'slowest_measured_execution_time'
+
+    def append(self, endpoint_call, dependencies):
+        if self.value < endpoint_call.execution_time:
+            self.value = endpoint_call.execution_time
+
+    def rendered_value(self):
+        return self.value
+
+
+class TDigestStatistic(Statistic):
+    dependencies = []
+
+    def __init__(self):
+        super().__init__()
+        self.value = self.empty()
+        self._digest = tdigest.TDigest()
+
+    def empty(self):
+        return -1
+
+    @abc.abstractmethod
```

Comment:
Is it really required to re-define an abstract method that the parent already defines? I don't think so.

Reply:
Fair point :+1: